extensions/amp-lightbox-viewer/0.1/service/lightbox-manager-impl.js

```diff
@@ -161,6 +161,22 @@ export class LightboxManager {
   }
 
   /**
+   * Return a list of lightboxable elements
+   * @return {!Promise<?Array<!Element>>}
+   */
+  getElements() {
+    return this.maybeInit_().then(() => {
+      if (!this.elements_) {
```

Comment:
`maybeInit_` already handles calling `scanLightboxables_` if needed. This can just:

```
return this. maybeInit_().then(() => this.elements_);
```